### PR TITLE
Added Routing + Categories and Image Component

### PR DESCRIPTION
**Pull request by evanugarte**
Please take a look at what i have added below.

NavBar:
image

Categories component (the images are black and i zoomed out on purpose)
image

If you click Visit, the image list component will show at the specified endpoint. see below.
image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
-import App from "./App";
+import { BrowserRouter, withRouter } from "react-router-dom";
+import Routing from "./Routing";
+import Navbar from "./Navbar";
+import "./App.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+function App(props) {
+
+  const [authenticated, setAuthenticated] = useState(true);
```
Comment (evanugarte) [inline src/index.js]: here is where you set if the user is authenticated or not. The routes will change as a result.

File: src/LoginPage.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Container, FormGroup, Label, Input, Button } from "reactstrap";
+
+function LoginPage() {
+
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const forms = [
+    { text: "Email", name: "email", callback: setEmail },
```
Comment (minhphan156) [inline src/LoginPage.js]: { text: "Email", name: "email", setEmail } is more clear

File: src/LoginPage.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    { text: "Password", name: "password", callback: setPassword }
```
Comment (minhphan156) [inline src/LoginPage.js]: { text: "Password", name: "password", setPassword } is more clear

File: src/LoginPage.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      <form onSubmit={(e) => { e.preventDefault(); }}>
+        {forms.map((x, index) => {
+          return (
+            <FormGroup key={index}>
+              <Label>{x.text}</Label>
+              <Input
+                autoFocus
+                name={x.name}
+                type={x.name}
+                onChange={(e) => x.callback(e.target.value)}
```
Comment (minhphan156) [inline src/LoginPage.js]: change x.callback according to the fix above

File: src/LoginPage.js (hunk 2 above)
Comment (evanugarte) [reply src/LoginPage.js]: Done.

File: src/LoginPage.js (hunk 3 above)
Comment (evanugarte) [reply src/LoginPage.js]: Done.

File: src/LoginPage.js (hunk 4 above)
Comment (evanugarte) [reply src/LoginPage.js]: Done.

**Pull request merged**